### DISCUSSION TITLE
transform to or add a nocode sqlite hosting

### DISCUSSION THREAD

**Discussion by Productivix** [sqlitebrowser/sqlitebrowser#3337] [Ideas]
hi, 
I study Nocodb with Sqlite databases for my customers, I connect my mobile aps to it via API .
I cannot find a Sqlite hosting with Nocodb server and I see that with small effort you have almost all : the nocode browser (to be changed in a server) and the sqlite hosting. What do you think ?

**Comment by justinclift** [comment]
Hi, that's a bit unclear (to me).

It *sounds* like you want some kind of SQLite server, that can be accessed via an API.

We already provide a server to do that (https://DBHub.io), and there is an API (https://api.dbhub.io).

The API uses JSON for communication, so as long as you can send + receive JSON, you should be able to use it.

Does that help? :smile:

**Comment by Productivix** [comment]
Hi, thank you , it seems to help a lot , yes, you understood well what I was looking for  - i dis not see the api . I will study that .
But in fact in my post there was a research, but also a suggestion for your project.

**Reply by justinclift** [reply]
Is your suggestion "the nocode browser"?

If that's correct, would you be ok to describe what you mean by that?

The words "nocode browser" don't mean anything to me by themselves.  So, I need more description in order to understand. :smile:

**Comment by justinclift** [comment]
@Productivix Aha.  I think this is what you mean: https://nocodb.com / https://github.com/nocodb/nocodb

That's an interesting project.  I've not seen that before.

It seems to be quite a powerful solution.

From looking through the code quickly, it seems to be written in NodeJS, and designed to be run in an intranet.  Personally, I wouldn't feel comfortable running it on an public internet site, at least not without becoming far more familiar with its code.

So, that's unlikely to happen any time soon. :wink:

**Comment by Productivix** [comment]
ok thanks
